src/Traits/ZaakTypeByIdentifier.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace OWC\Zaaksysteem\Traits;
+
+use OWC\Zaaksysteem\Contracts\Client;
+use OWC\Zaaksysteem\Endpoints\Filter\ResultaattypenFilter;
+use OWC\Zaaksysteem\Entities\Zaaktype;
+use OWC\Zaaksysteem\Support\Collection;
+
+trait ZaakTypeByIdentifier
```

Comment:
Hier hetzelfde als bij `ResolveBSN`. Al twijfel ik hier over de oplossing. Wellicht integreren in de `ZaaktypenEndpoint` class? Dit probleem speelt natuurlijk bij alle clients (openzaak, decos, rx, etc.).

Reply:
Nieuwe method in het Zaaktype endpoint voor gemaakt.